recipes/shortstack/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "ShortStack" %}
+{% set version = "3.8.3" %}
+{% set sha256 = "7c76f51ed949ca95ec5df7cb54803ae2350658fd64c75909351d5a856abb0dbe" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/MikeAxtell/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - perl 5.22.0.1
```

Comment:
please don't pin perl if not really necessary.

Reply:
The program is a `perl` script:
https://github.com/MikeAxtell/ShortStack/blob/master/ShortStack

Reply:
Yes, but you don't need to pin the version, isn't it?

Reply:
I had issues when leaving the `perl` version unspecified... can try to reproduce it to check